package.json

```diff
@@ -23,20 +23,22 @@
   "author": "mhlabs",
   "license": "ISC",
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.87.0",
-    "@aws-sdk/lib-dynamodb": "^3.87.0"
+    "@aws-sdk/client-dynamodb": "^3.165.0",
+    "@aws-sdk/lib-dynamodb": "^3.165.0"
   },
   "devDependencies": {
-    "@types/jest": "^27.5.0",
-    "aws-sdk-client-mock": "^0.6.2",
-    "eslint": "^8.15.0",
-    "eslint-config-airbnb-base": "15.0.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^26.1.5",
-    "eslint-plugin-prettier": "^4.0.0",
-    "jest": "^28.1.0",
-    "prettier": "^2.6.2"
+    "@tsconfig/node16": "^1.0.3",
+    "@types/jest": "^27.4.1",
+    "aws-sdk-client-mock": "^2.0.0",
+    "esbuild": "^0.15.7",
+    "esbuild-jest": "^0.5.0",
```

Comment:
esbuild-jest not used anymore right?

Reply:
I thought I removed this :O Will remove again